docs/source/notes/get_start_xpu.rst

```diff
@@ -1,109 +1,81 @@
-Pytorch 2.4: Getting Started on Intel GPU
-=========================================
+Getting Started on Intel GPU
+============================
 
-The support for Intel GPUs is released alongside PyTorch v2.4.
-
-This release only supports build from source for Intel GPUs.
-
-Hardware Prerequisites
-----------------------
+Hardware Prerequisite
+---------------------
 
 .. list-table::
+   :widths: 50 50
    :header-rows: 1
 
-   * - Supported Hardware
-     - Intel® Data Center GPU Max Series
-   * - Supported OS
+   * - Validated Hardware
+     - Supported OS
+   * - Intel® Data Center GPU Max Series
      - Linux
+   * - Intel Client GPU
+     - Windows/Linux
+
+Intel GPUs support enhancement is ready in PyTorch* 2.5 to provide Beta quality for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios, especially in AIPC scene.
 
+Software Prerequisite
+---------------------
 
-PyTorch for Intel GPUs is compatible with Intel® Data Center GPU Max Series and only supports OS Linux with release 2.4.
+Visit `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_ for more detailed information regarding:
 
-Software Prerequisites
-----------------------
+#. Intel GPU driver installation
+#. Intel support package installation
+#. Environment setup
 
-As a prerequisite, install the driver and required packages by following the `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_.
+Installation
+------------
 
-Set up Environment
-------------------
+Binaries
+^^^^^^^^
+
+Platform Linux
+""""""""""""""
 
-Before you begin, you need to set up the environment. This can be done by sourcing the ``setvars.sh`` script provided by the ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages.
+
+Now we have all the required packages installed and environment activated. Use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` on Linux.
 
 .. code-block::
 
-   source ${ONEAPI_ROOT}/setvars.sh
+    pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/xpu
```

Comment:
I get HTTP/403 when trying to access https://download.pytorch.org/whl/xpu

Reply:
Hi @malfet , for now we use pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu, the link in the blog is official one which will be available later.

Reply:
I'm not sure this is the case, but sure,  if you insist on pointing the users to broken link, this is your choice.